dev/src/constants/Configurations.ts

```diff
@@ -9,8 +9,39 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
-export enum CWConfigurations {
-    AUTO_SHOW_VIEW = "codewind.autoShowView",
-    OVERVIEW_ON_CREATION = "codewind.openOverviewOnCreation",
-    ALWAYS_CREATE_IN_WORKSPACE = "codewind.alwaysCreateProjectsInWorkspace",
+/*******************************************************************************
```

Comment:
With this PR, there's two copyright headers in the file -- one should be removed.